tests/x11/reboot_gnome.pm

```diff
@@ -19,7 +19,7 @@ use utils;
 sub run {
     my ($self) = @_;
     # 'keepconsole => 1' is workaround for bsc#1044072
-    power_action('reboot', keepconsole => 1);
+    power_action('reboot', keepconsole => 1, s390_reboot => get_var('S390_ZKVM', 0));
```

Comment:
Do we need the explicit `0` here? `power_action()` seems to gate it sufficiently.

Reply:
@mnowaksuse, this would allow us to control the behavior by setting, without changing code.

Or do you mean the exlicit 0 for get_var()?

Reply:
Sorry for not making it clear. Yes, I meant the `0` in `get_var('S390_ZKVM', 0)`.